scripts/vote_2022_06_14.py

```diff
@@ -0,0 +1,105 @@
+"""
+Voting 14/06/2022.
+
+1. Revoke DEPOSIT_ROLE from old DepositSecurityModule 0xDb149235B6F40dC08810AA69869783Be101790e7
+2. Grant DEPOSIT_ROLE to new DepositSecurityModule 0x710B3303fB508a84F10793c1106e32bE873C24cd
+3. Set lastDepositBlock of DepositSecurityModule to {{??? block number TBD}}
+
```

Comment:
We may need to add a missing point: '4. Set IPFS contents of the Lido app to ...'